com.microsoft.mrtk.uxcomponents/ObjectBar/HorizontalAppBar.prefab

```diff
@@ -181,6 +181,22 @@ PrefabInstance:
       propertyPath: m_Sprite
       value: 
       objectReference: {fileID: 21300000, guid: e7a37761780007545a89ece9b43ce0e8, type: 3}
+    - target: {fileID: 6400998762086782788, guid: e5b53b02ddc453d49a3a36a85164095f, type: 3}
```

Comment:
Is there a reason for these changes, too?

Reply:
Reverted this unnecessary change as well.